requirements.txt

```diff
@@ -0,0 +1,58 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
```

Comment:
Please provide more generic requirements.
Only the minimal (& top-level) packages required by this tool should be added. 
Strict definitions will make it hard to install it within other environments that would not match specifically those versions.